torch_np/_ndarray.py

```diff
@@ -362,6 +362,7 @@ def reshape(self, *shape, order="C"):
 
     diagonal = _funcs.diagonal
     trace = _funcs.trace
+    dot = _funcs.dot
```

Comment:
missing also `vdot`?

Reply:
Missing in numpy, yes :-)

In [23]: hasattr(np.array([1, 2, 3]), 'vdot')
Out[23]: False

Reply:
lol. Let's still add it to be forward-looking.

Reply:
OK, can do. 
If we're starting to do this, what's the guiding principle, where do we stop however. E.g. do we want feature parity for the main namespace and ndarray methods?

```
In [24]: len(dir(np))
Out[24]: 595

In [25]: len(dir(np.ndarray))
Out[25]: 165
```

Reply:
If we can do it "for free" it may be a fine thing to do? WDYT @rgommers?

Reply:
There is no guiding principle, it's an accident of history. It's highly unlikely though that we'll add more method to `numpy.ndarray`, there are already too many. So I wouldn't add anything that's not already present.